lib/foreman_remote_execution/engine.rb

```diff
@@ -244,6 +244,12 @@ class Engine < ::Rails::Engine
           parent: :monitor_menu,
           after: :audits
 
+        menu :labs_menu, :job_invocations_detail,
+          url_hash: { controller: :job_invocations_detail, action: :index },
+          caption: N_('Job invocations detail'),
+          parent: :lab_features_menu,
+          url: '/experimental/job_invocations_detail/1'
```

Comment:
This is a bit unfortunate as it points at a particular resource which may or may not exist. How about we modify the old details page to include the link to the new one if the experimental page setting is enabled?

Reply:
I was planning to move it from the Lab features after some main functionality is done, but I like your solution.